packages/terra-form/tests/nightwatch/number-field/number-field-spec.js

```diff
@@ -0,0 +1,48 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+
+const screenshot = require('terra-toolkit').screenshot;
+
+module.exports = {
+  before: (browser, done) => {
+    browser.resizeWindow(browser.globals.width, browser.globals.height, done);
+  },
+
+  afterEach: (browser, done) => {
+    screenshot(browser, 'terra-form', done);
+  },
+
+  'Displays a default NumberField with a number input': (browser) => {
+    browser
+      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/form-tests/number-field/default`)
+      .assert.elementPresent('.terra-Form-field')
+      .assert.elementPresent('input[type="number"]');
+  },
+
+  'Displays a populated NumberField with correct label, help message, and choices': (browser) => {
```

Comment:
Can we break this into two tests or update the description? This is testing for inline and correct field population